tests/app/config/Shared/config_default.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Copyright © 2021-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+use Spryker\Shared\Propel\PropelConstants;
+use Spryker\Zed\Propel\PropelConfig;
+
+$config[PropelConstants::ZED_DB_ENGINE]
+    = strtolower(getenv('SPRYKER_DB_ENGINE') ?: '') ?: PropelConfig::DB_ENGINE_MYSQL;
```

Comment:
Config value should not be required as soon as [TransferGenerateHelper fix](https://github.com/spryker/spryker/pull/8976) is deployed